public/css/signup.css

```diff
@@ -0,0 +1,41 @@
+.signup{
+    display: flex;
+    /* padding-left: 4rem; */
+   
+    align-content: center;
+}
+
+
+
+form{
+    background-color:#E6E9F8 ;
+    width: 60%;
+    /* border: 1px solid red; */
+    gap: 2rem;
+    padding: 1rem;
+    padding-top: 4rem;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    
+}
+input{
+   height: 2rem;
+   padding-left: 1rem;
+   padding: 0.5rem;
+
+}
+#btn{
+    
+    color: blanchedalmond;
+    padding: 0.5rem;
+    width: 60%;
+
+}
+.FM{
+   
+    border: none;
+    width: 18rem;
```

Comment:
On the phone, the input field will be (16x18) px and the form will be max 60% of 390px, for example.
![image](https://user-images.githubusercontent.com/24773276/235850805-f7df748e-052a-46c1-b8a1-2a623f7dc42f.png)